archivy/cli.py

```diff
@@ -22,20 +26,77 @@ def cli():
 
 
 # add built in commands:
-cli.add_command(routes_command)
 cli.add_command(shell_command)
 
 
+@cli.command("init", short_help="Initialise your archivy application")
+@click.pass_context
+def init(ctx):
+    try:
+        load_config()
+        click.confirm("Config already found. Do you wish to reset it? "
+                      "Otherwise run `archivy config`", abort=True)
+    except FileNotFoundError:
+        pass
+
+    config = Config()
+    delattr(config, "SECRET_KEY")
+
+    click.echo("This is the archivy installation initialization wizard.")
+    set_curr_dir = click.confirm("Use current directory as data directory for archivy?")
+    if set_curr_dir:
+        data_dir = os.getcwd()
+    else:
+        data_dir = click.prompt("Otherwise, enter the full path of the "
+                                "directory where you'd like us to store data.", type=str)
+
+    es_enabled = click.confirm("Would you like to enable Elasticsearch? For this to work "
+                               "when you run archivy, you must have ES installed.")
+    if es_enabled:
+        config.ELASTICSEARCH_CONF["enabled"] = 1
+    else:
+        delattr(config, "ELASTICSEARCH_CONF")
+
```

Comment:
It's mentioned in the docs and the command that it must be installed for that feature, otherwise I'd rather leave it to the user if they install it before / after configuration.

Reply:
I wonder how many people read docs

Reply:
I mean we say in the command:

```
Would you like to enable Elasticsearch? For this to work when you run archivy, you must have ES installed.
```

Reply:
yea, thats perhaps better than what it is now. Maybe, it would be even better if you add a link to the elastic search part when you ask this, so people would be informed about what it is too..
A link to archivy's elasticsearch doc would be more than enough :smile:

Reply:
Good idea! Btw that line is what I've already put in the code, but I think you're right linking it would be nice.